bokeh/__init__.py

```diff
@@ -10,10 +14,9 @@
     __version__ = get_versions()['version']
     del get_versions
 
-
 _notebook_loaded = None
 
-def load_notebook(resources=None, verbose=False, force=False, skip=False):
+def load_notebook(resources=None, verbose=False, skip=False):
```

Comment:
`skip` doesn't actually appear to get used? Maybe use `skip` below to omit call to  `load_notebook` altogether, and remove it as a param?